qiskit/providers/ibmq/job/job_monitor.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""A module for monitoring various qiskit functionality"""
+
+import sys
+import time
+import datetime
+from typing import TextIO
+from .ibmqjob import IBMQJob
+from ..utils.converters import seconds_to_duration
+
+
+def _text_checker(job: IBMQJob,
+                  interval: float,
+                  _interval_set: bool = False,
+                  quiet: bool = False,
+                  output: TextIO = sys.stdout) -> None:
+    """A text-based job status checker.
+
+    Args:
+        job: The job to check.
+        interval: The interval at which to check.
+        _interval_set (bool): Was interval time set by user?
+        quiet (bool): If True, do not print status messages.
```

Comment:
Why is this parameter necessary? If `quiet=True`, then this function is essentially the same as `wait_for_final_state` as it just waits for the job to finish without printing anything.

Reply:
Quiet was added here: https://github.com/Qiskit/qiskit-terra/pull/2037.  I agree it is not needed anymore.